src/components/CookieBar/CookieBar.js

```diff
@@ -1,37 +1,15 @@
 import React, { useState } from 'react';
 import { CookieModal } from 'hds-react';
-import { enableMatomoTracking } from "../../utils/cookieUtils";
+import { getHDSCookieConfig } from "../../utils/cookieUtils";
 import config from '../../config';
 
 function CookieBar() {
+  const siteName = document.querySelector("meta[property='og:title']").getAttribute('content');
   const [language, setLanguage] = useState(config.activeLanguage);
-
   const getCookieModalConfig = () => {
-    return {
-      siteName: document.querySelector("meta[property='og:title']").getAttribute('content'),
-      currentLanguage: language,
-      optionalCookies: {
-        groups: [
-          {
-            commonGroup: 'statistics',
-            cookies: [{
-              commonCookie: 'matomo'
-            }],
-          },
-        ],
-      },
-      language: {
-        onLanguageChange: setLanguage,
-      },
-      focusTargetSelector: '#focused-element-after-cookie-consent-closed',
-      onAllConsentsGiven: (consents) => {
-        // called when consents are saved
-        // handle changes like:
-        if (consents.matomo) {
-          enableMatomoTracking();
-        }
-      },
-    };
+    const config = getHDSCookieConfig(siteName, language, setLanguage);
+    return config;
+    // return getHDSCookieConfig(siteName, language, setLanguage);
```

Comment:
Could return `getHDSCookieConfig(siteName, language, setLanguage);` straight away. Also commented code could be removed

Reply:
For unknown reason when trying to return it directly it didn't work... :)

Reply:
Lol, well the commented code could still be removed 😁

Reply:
That has now been done and committed.